src/eval/operation.rs

```diff
@@ -1974,7 +1983,7 @@ impl<R: ImportResolver, C: Cache> VirtualMachine<R, C> {
                         } else {
                             Err(EvalError::TypeError(
                                 String::from("Record"),
-                                String::from("insert"),
+                                String::from("record_insert expects its second argument to be a Record"),
```

Comment:
The error can't actually be triggered by `record.insert` because that has a contract check intervening. So the only way to trigger this is really to use the `%record_insert%` primop directly.

Reply:
Oh yeah, sorry, I think I meant to comment on the error that says that the field is already present.

Reply:
So, we agreed that this particular case should be addressed by a judicious use of contract annotations in the standard library. The end goal should be to keep the user from seeing these primop errors unless they used a primop explicitly.